### PR TITLE
Add git autoupdate config in springy

### PR DESCRIPTION
**Pull request by Piicksarn**

### CODE REVIEW

**Comment by Piicksarn** [comment]
@LeaYeh would you please cheacking this commit for me?
This commit is for updating the lib springy, which can not autoupdate.

**Comment by LeaYeh** [comment]
why didn't you add auto update config directly?

**Comment by Piicksarn** [comment]
https://cdnjs.com/libraries/springy
it should update by manual.

**Comment by Piicksarn** [comment]
@LeaYeh ~so ...is there any problems in this commit :D?~

**Comment by Piicksarn** [comment]
@LeaYeh ~ Sorry about the mistake!
I have it corrected, and would you please checking the commit ?
Thank you!

**Comment by LeaYeh** [comment]
your commit content is not match  the pr.

**Comment by Piicksarn** [comment]
@LeaYeh ~I have it corrected!

File: ajax/libs/springy/package.json [hunk 1]
```diff
@@ -12,5 +12,14 @@
...
     "email": "dennis.hotson@gmail.com",
     "url": "http://dhotson.tumblr.com"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/dhotson/springy.git",
+    "basePath": "",
+    "files": [
+      "springy.js",
+      "springyui.js"
```
Comment (LeaYeh) [inline ajax/libs/springy/package.json]: please check it again.

Comment (Piicksarn) [inline ajax/libs/springy/package.json]: Do i need to change the file to springy*.js?
It is the only part I found...

**Comment by LeaYeh** [comment]
yep you got the point, try to fix it :)

**Comment by Piicksarn** [comment]
@LeaYeh ,I have it corrected, and need your help for checking the PR.
Thank you!

**Comment by LeaYeh** [comment]
hmmmmm... different structure.

**Comment by Piicksarn** [comment]
@LeaYeh, after i saw the document below, i think it's fine in my file map
https://github.com/isaacs/node-glob

**Comment by LeaYeh** [comment]
which part you mean in doc?

**Comment by Piicksarn** [comment]
@LeaYeh, would you please checkout the pr again?
Thank you!

**Comment by Piicksarn** [comment]
@LeaYeh

**Comment by LeaYeh** [comment]
i think it ok. @PeterDaveHello

**Pull request merged**